core/git_command.py

```diff
@@ -31,6 +31,22 @@ class GitSavvyError(Exception):
     pass
 
 
+class IsolatedGitCommand(GitCommand):
+    """
+    Allow accessing GitCommand outside the scope of a WindowCommand or
+    TextCommand.  You must manually pass in a valid repository path.
+    """
+    def __init__(self, repo_path=None):
+        self._repo_path = repo_path
```

Comment:
Are you sure?  Because this is a subclass of GitCommand I needed to override it's behavior for retrieving repo_path... does just setting a static value kill the property method from the parent class?